apps/rush-lib/src/logic/PublishUtilities.ts

```diff
@@ -642,25 +642,32 @@ export class PublishUtilities {
       dependencies[change.packageName] &&
       !PublishUtilities._isCyclicDependency(allPackages, parentPackageName, change.packageName)
     ) {
-      const requiredVersion: string = new DependencySpecifier(
+      const requiredVersion: DependencySpecifier = new DependencySpecifier(
         change.packageName,
         dependencies[change.packageName]
-      ).versionSpecifier;
+      );
+      const isAnyWorkspaceVersion: boolean =
```

Comment:
"AnyWorkspace" is a little confusing. It could be interpreted to mean "is it any variety of a workspace version," instead of "a workspace version that specifically selects anything."

Perhaps we should call this `isWorkspaceWildcardVersion`

Reply:
Agreed